providers/hexonet/hexonetProvider.go

```diff
@@ -0,0 +1,77 @@
+// Package hexonet implements a registrar that uses the hexonet api to set name servers. It will self register it's providers when imported.
+package hexonet
+
+// HEXONET is a leading developer and operator of domain names and DNS platforms.
+// Thousands of individuals, service providers and registrars globally choose HEXONET
+// for domains and DNS for our advanced technology, operational performance and up-time,
+// and most importantly for unparalleled expertise in DNS.
+// DnsControl with HEXONET's DNS marries DNS automation with an industry-leading DNS
+// platform that supports DNSSEC, PremiumDNS via Anycast Network, and nearly all of
+// DnsControl's listed provider features.
+
+import (
+	"encoding/json"
+
+	"github.com/StackExchange/dnscontrol/providers"
+	hxcl "github.com/hexonet/go-sdk/client"
+	"github.com/pkg/errors"
+)
+
+// HXClient describes a connection to the hexonet API.
+type HXClient struct {
+	APILogin    string
+	APIPassword string
+	APIEntity   string
+	client      *hxcl.Client
+}
+
+var features = providers.DocumentationNotes{
+	providers.CanUseAlias:            providers.Can("We fallback to CNAME up to now. Our API comes already with realtime ALIAS support over X-ALIAS-A / X-ALIAS-AAAA. This will be supported shortly."),
+	providers.CanUseCAA:              providers.Can(),
+	providers.CanUsePTR:              providers.Can(),
+	providers.CanUseRoute53Alias:     providers.Cannot(),
+	providers.CanUseSRV:              providers.Can(),
+	providers.CanUseTLSA:             providers.Can(),
+	providers.CanUseTXTMulti:         providers.Can(),
+	providers.CantUseNOPURGE:         providers.Can(),
+	providers.DocCreateDomains:       providers.Can(),
+	providers.DocDualHost:            providers.Can(),
+	providers.DocOfficiallySupported: providers.Can(),
```

Comment:
"Officially supported" is a designation that means the maintainers (Stack Exchange) actively use and guarantee support for a provider. The hexonet provider will be "community supported", as we do not have the expertise to debug it thoroughly. But you do :)

Reply:
"officially supported" providers block a release if they don't work. "community supported" is best effort... we'll ship a release if it is broken and haven't heard back from the author.

Reply:
Ok, I will forward this - I was asked to get us as officially supported. If that's not possible atm, I can't change it :)

Reply:
got already changed to community supported.